src/main/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+spring.data.web.pageable.size-parameter=size
```

Comment:
Opinion on `.properties` vs `.yml` file for config?

Reply:
No personal preference, but I've read that there might be a problem with `.yml` if we ever need `@PropertySource` annotations... But if you prefer `.yml`, we can switch (better sooner than later).

Reply:
Understood. No issues with it, so let's leave it for now.